examples/cart/index.js

```diff
@@ -118,39 +118,58 @@ $(function() {
     $('.buy-button').on('click', function (event) {
       event.preventDefault();
       var id = product.selectedVariant.id;
-      addVariantToCart(product.selectedVariant, 1);
+      addOrUpdateVariant(product.selectedVariant, 1);
+      setPreviousFocusItem(this);
+      $('#checkout').focus();
     });
   }
 
   /* Increase product variant quantity in cart
   ============================================================ */
   function attachQuantityIncrementListeners(product) {
-    $('.cart').on('click', '.quantity-increment', function() {
+    $('.cart').on('click', '.quantity-increment', function(evt) {
       var variantId = parseInt($(this).attr('data-variant-id'), 10);
       var variant = product.variants.filter(function (variant) {
         return (variant.id === variantId);
       })[0];
 
-      $(this).closest('.cart-item').addClass('js-working');
-      $(this).attr('disabled', 'disabled');
-
-      addVariantToCart(variant, 1);
+      addOrUpdateVariant(variant, 1);
     });
   }
 
   /* Decrease product variant quantity in cart
   ============================================================ */
   function attachQuantityDecrementListeners(product) {
-    $('.cart').on('click', '.quantity-decrement', function() {
+    $('.cart').on('click', '.quantity-decrement', function(evt) {
       var variantId = parseInt($(this).attr('data-variant-id'), 10);
       var variant = product.variants.filter(function (variant) {
         return (variant.id === variantId);
       })[0];
 
-      $(this).closest('.cart-item').addClass('js-working');
-      $(this).attr('disabled', 'disabled');
+      addOrUpdateVariant(variant, -1);
+    });
+  }
+
+  /* Bind Event Listeners
+  ============================================================ */
+  function bindEventListeners() {
+    $('.cart .btn--close').on('click', closeCart);
+
+    $(document).on('click', function(evt) {
+      if((!$(evt.target).closest('.cart').length) && (!$(evt.target).closest('.js-preventCartListener').length)) {
+        closeCart();
+      }
+    });
 
-      addVariantToCart(variant, -1);
+    var ESCAPE_KEYCODE = 27
```

Comment:
This should live up near the top where you define any other globals, constants, whatevs.